PendingReleaseNotes

```diff
@@ -147,6 +147,14 @@
   `s3:GetObjectRetention` are also considered when fetching the source object.
   Replication of tags is controlled by the `s3:GetObject(Version)Tagging` permission.
 
+* RADOS: A new command, `ceph osd pool availability-status`, has been added that allows
```

Comment:
Should I mention that the feature is in tech preview in release notes as well?

Reply:
I think so yes,

Reply:
Yeah I think you could